software/cloud-dashboard/node-server/config/device-map.js

```diff
@@ -6,7 +6,7 @@
 module.exports={
 	locMap : {
 	"4b0031000d51353432393339": "ADPLKenyaC6304",
-        "300057000951343334363138": "ADPLKenyaN3763",
+        "3e001d000e51353432393339": "ADPLKenyaN6833",
```

Comment:
I can go ahead and make the change in a separate diff or you can just make it in this one and then I'll merge to master.

If we just want to get this out--if you just keep the name `"ADPLKenyaN3763"` I can merge this to master and deploy it so we don't miss data :)